development/Models/decisionTree.py

```diff
@@ -0,0 +1,53 @@
+# Imports
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+from sklearn.tree import plot_tree
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.model_selection import train_test_split
+from sklearn.model_selection import GridSearchCV
+from sklearn.metrics import accuracy_score
+from sklearn.metrics import classification_report
+from sklearn.metrics import confusion_matrix
+from sklearn.metrics import f1_score
+
+# Read csv, StaterData
+df = pd.read_csv('/Users/laurensheberle/Github/StaterData.csv')
+
+# Define your features and target, with x as text_column which contains the complaint in the dataset. And target_column which contains the target variable
+X = df['Clean consumer complaint']  
+y = df['Issue']                        
```

Comment:
Unnecessary white spaces after df['Issue']

Reply:
In the notebook there are some unnecessary white spaces. In the notebook there are multiple cells that have excessive white spaces in the end.
![image](https://github.com/IT-Guy007/Stater-text-classification/assets/74193681/ca8195fb-ae9f-46ea-b78c-82394e4014bd)
![image](https://github.com/IT-Guy007/Stater-text-classification/assets/74193681/7ca44828-d0c3-4e82-aa09-761adc64449f)